src/app/modules/assets/components/Assets.js

```diff
@@ -28,7 +28,7 @@ export default class Assets extends Component {
       const localeFile = Object.keys(file)[0]
       const localeTitle = Object.keys(title)[0]
       const localizedTitle = title[localeTitle]
-      const ext = /\.(.*)$/.exec(file[localeFile].fileName)[1]
+      const ext = file[localeFile].fileName.split('.').slice(1).join('') || "no extension"
```

Comment:
Hey @cachrisman this won't work if you have a filename like `my.example.file.js`
a better approach would be 
```js
const fileName = file[localeFile].fileName
const ext = fileName.slice((fileName.lastIndexOf(".") - 1 >>> 0) + 2)
```

Reply:
☝️